keepa/_version.py

```diff
@@ -1,6 +1,6 @@
 """Version number for keepa
 """
 # major, minor, patch, -extra
-version_info = 1, 1, 0
+version_info = 1, 1, 0, 1
```

Comment:
While I appreciate the reminder to bump the version, the `-extra` flag usually indicates a release candidate and is non-numeric.  See: https://semver.org/

Regardless, we're badly in need of a version bump, and I'll bump this and #75 are merged.